client/client_test.go

```diff
@@ -2732,6 +2790,14 @@ func TestRotateKeyAfterPublishServerManagementChange(t *testing.T) {
 		data.CanonicalTargetsRole:  false,
 		data.CanonicalRootRole:     false,
 	})
+	// check that the snapshot remote rotation creates new keys
```

Comment:
non-blocking: I think this snapshot case may already be tested in the previous batch?  Although it rotates all those keys at once, as opposed to one at a time.  Possibly we can just add the timestamp role to the list of keys to rotate in these tests?